### PR TITLE
Do not show avatar transition on signup

### LINKED ISSUE #6483: Do not show avatar transition on signup
It seemed like animating/transitioning in the new avatar on click of the "Create account" button. Turns out this introduces more problems than it is worth. Let's just show the relevant avatar by default with no fancy business.

This will also resolve https://bugzilla.mozilla.org/show_bug.cgi?id=1614273#c2

### PR DESCRIPTION
**Pull request by schalkneethling**
Fixes #6483 

Tested with and without multi auth enabled.

### CODE REVIEW

**Comment by schalkneethling** [comment]
@peterbe This is just so weird. Why are we seeing such different things? I do not see the newsletter sign-up checkbox at all, is that a `.env` setting?

Also, the avatar not loading, what is the `src` of the HTML element?

**Comment by schalkneethling** [comment]
Here is what I see:

create-account

**Comment by schalkneethling** [comment]
I am going to open an issue to just not bother with that second check-mark. It makes things way too complicated for very little(if any) user benefit. But that is a separate issue with a separate PR :)

**Comment by peterbe** [comment]
> Also, the avatar not loading, what is the `src` of the HTML element?

`undefined`.

**Comment by schalkneethling** [comment]
@peterbe I also notice that the Google logo in your screenshot is grayed out. That is not implemented yet in this PR so, perhaps you need to `docker-compose exec web make build-static`?

**Comment by schalkneethling** [comment]
> > Also, the avatar not loading, what is the `src` of the HTML element?
> 
> `undefined`.

Hmmm, so for some reason for you `{{ account.get_avatar_url() }}` returns `undefined`.

**Comment by schalkneethling** [comment]
> > > Also, the avatar not loading, what is the `src` of the HTML element?
> > 
> > 
> > `undefined`.
> 
> Hmmm, so for some reason for you `{{ account.get_avatar_url() }}` returns `undefined`.

This makes me think it is JS trying to set the `src` and not finding the `data-` attribute.

File: kuma/users/jinja2/socialaccount/signup.html [hunk 1]
```diff
@@ -37,7 +37,7 @@ <h1>{{ _('Create your MDN account') }}</h1>
         </li>
       {% endif %}
       <li class="mdn-profile">
-        <img src="{{ static('auth/mdn-auth.svg') }}" width="90px" alt="{{ _('User avatar') }}" />
+        <img src="{{ account.get_avatar_url() }}" width="90px" alt="{{ _('User avatar') }}" />
```
Comment (peterbe) [inline kuma/users/jinja2/socialaccount/signup.html]: Perhaps I'm old but I learned back in the ancient days of decades ago that the `width` was just a number. Can you use `px` in them these days?

Comment (schalkneethling) [reply kuma/users/jinja2/socialaccount/signup.html]: Ha! weird. It will probably just be ignored. Not sure why it has `px`

**Pull request merged**